### PR TITLE
add slide-toggle scene

### PR DESCRIPTION
**Pull request by annieyw**
First draft:
image

### CODE REVIEW

File: scenes/src/app/scenes/slide-toggle/slide-toggle-scene.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.wrapper {
```
Comment (jelbourn) [inline scenes/src/app/scenes/slide-toggle/slide-toggle-scene.scss:1]: We should prefix all CSS selectors, either with `docs-` or with `scene-`

(in this and other PRs)

**Comment by mmalerba** [comment]
I wonder if there's one more toggle we could add, it looks a little sparse

**Comment by annieyw** [comment]
image

added wifi toggle

File: scenes/src/app/scenes/slide-toggle/slide-toggle-scene.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.scene-slide-toggle {
+  margin: 15px;
+  width: 250px;
+}
+
+mat-slide-toggle {
+  float: right;
```
Comment (mmalerba) [inline scenes/src/app/scenes/slide-toggle/slide-toggle-scene.scss:7]: don't use `float`

File: scenes/src/app/scenes/slide-toggle/slide-toggle-scene.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'app-slide-toggle-scene',
+  templateUrl: './slide-toggle-scene.html',
+  styleUrls: ['./slide-toggle-scene.scss']
+})
+export class SlideToggleScene {
+
```
Comment (mmalerba) [inline scenes/src/app/scenes/slide-toggle/slide-toggle-scene.ts:13]: nit: extra lines

**Pull request merged**